elements/pfe-label/pfe-label.ts

```diff
@@ -1,95 +1,102 @@
+ * @cssprop {<color>} --pf-c-label--m-outline--m-orange__content--Color     {@default `#8f4700`}
+ *
+ * @cssprop {<color>} --pf-c-label--m-purple__content--Color                {@default `#1f0066`}
+ * @cssprop {<color>} --pf-c-label--m-purple--BackgroundColor               {@default `#f2f0fc`}
+ * @cssprop {<color>} --pf-c-label--m-purple__content--before--BorderColor  {@default `#cbc1ff`}
+ * @cssprop {<color>} --pf-c-label--m-outline--m-purple__content--Color     {@default `#6753ac`}
+ *
+ * @cssprop {<color>} --pf-c-label--m-red__content--Color                {@default `#7d1007`}
+ * @cssprop {<color>} --pf-c-label--m-red--BackgroundColor               {@default `#faeae8`}
+ * @cssprop {<color>} --pf-c-label--m-red__content--before--BorderColor  {@default `#c9190b`}
+ * @cssprop {<color>} --pf-c-label--m-outline--m-red__content--Color     {@default `#c9190b`}
+ *
+ * @cssprop {<color>} --pf-c-label--m-gold__content--Color                {@default `#3d2c00`}
+ * @cssprop {<color>} --pf-c-label--m-gold--BackgroundColor               {@default `#fdf7e7`}
+ * @cssprop {<color>} --pf-c-label--m-gold__content--before--BorderColor  {@default `#f9e0a2`}
+ * @cssprop {<color>} --pf-c-label--m-outline--m-gold__content--Color     {@default `#795600`}
+
+ * @cssprop {<color>} --pf-c-label--m-blue__icon--Color {@default `#06c`}
+ * @cssprop {<color>} --pf-c-label--m-cyan__icon--Color {@default `#009596`}
+ * @cssprop {<color>} --pf-c-label--m-green__icon--Color {@default `#3e8635`}
+ * @cssprop {<color>} --pf-c-label--m-orange__icon--Color {@default `#ec7a08`}
+ * @cssprop {<color>} --pf-c-label--m-red__icon--Color {@default `#c9190b`}
+ * @cssprop {<color>} --pf-c-label--m-gold__icon--Color {@default `#f0ab00`}
+ *
+ * @slot icon
+ *       Contains the labels's icon, e.g. web-icon-alert-success.
+ *
+ * @slot
+ *       Must contain the text for the label.
+ *
+ * @cssprop {<length>} --pf-c-label--m-compact--PaddingTop     {@default `0`}
+ * @cssprop {<length>} --pf-c-label--m-compact--PaddingRight   {@default `0.5rem`}
+ * @cssprop {<length>} --pf-c-label--m-compact--PaddingBottom  {@default `0`}
+ * @cssprop {<length>} --pf-c-label--m-compact--PaddingLeft    {@default `0.5rem`}
  */
-@customElement('pfe-label') @pfelement()
-export class PfeLabel extends LitElement {
+@customElement('pfe-label')
+export class PfeLabel extends BaseLabel {
   static readonly version = '{{version}}';
 
-  static readonly styles = [styles];
-
-  /**
-   * Sets a value for color for the label
-   * Options include grey, blue, green, orange, red, purple, cyan
-   * @default grey
-   */
-  @property({ reflect: true }) color?: 'grey' | 'blue' | 'green' | 'orange' | 'red' | 'purple' | 'cyan' = 'grey';
+  static readonly styles = [...BaseLabel.styles, styles];
 
-  /**
-  * Sets the value for a icon
-  */
-  @property({ reflect: true }) icon?: string | null = null;
+  @property({ reflect: true, type: Boolean, attribute: 'is-compact' }) isCompact = false;
```

Comment:
```suggestion
  @property({ reflect: true, type: Boolean }) compact = false;
```

[guide](https://github.com/RedHat-UX/red-hat-design-system/wiki/Custom-Elements-API-Style-Guide#host-attributes). True, PF react uses `isFoo`, but I think we're justified here in diverging slightly for the sake of clarity, and this is how I approached it for button.